### PR TITLE
Document friendly leak rates

### PR DESCRIPTION
**Pull request by fatalbanana**

### CODE REVIEW

File: doc/modules/ratelimit.md [hunk 1]
```diff
@@ -37,22 +43,22 @@ Where `type` is one of:
 Both these attributes are floating point values.
 
 From version `1.5` it is also possible to define limits in a simplified form:
-  bounce_to = "2 / 5m";
-
-Where the first number is number of messages and the second is timeframe. For example, the line above defines a limit for bounce messages per recipient at level 2 message in 5 minutes.
+`bounce_to = [100, "2 / 5m"];`
```
Comment (vstakhov) [inline doc/modules/ratelimit.md]: I don't understand your example. It just doesn't look right.

Comment (fatalbanana) [inline doc/modules/ratelimit.md]: It's `[bucket_size, leak_rate]`.

Comment (vstakhov) [inline doc/modules/ratelimit.md]: But why do we need size in this case? It is illogical, since we already have the size: namely, 2 in this case.

Comment (fatalbanana) [inline doc/modules/ratelimit.md]: Hmmm, maybe I miss something. My understanding: "2" (per 5 minutes) - ie. 0.0033333333 - is just the rate at which the bucket leaks and the size of the bucket still needs to be set independently- so we know when it's full.

**Comment by fatalbanana** [comment]
I've updated this (and the ratelimit module itself) per discussion.

**Pull request merged**